todolist.html

```diff
@@ -32,5 +32,60 @@
                 <i class='icon icon--left fa fa-trash-o'></i> Delete All
             </button>
         </div>
+        <div>
+            <div></div><!-- form overlay (functions same as modal overlay) -->
+            <p>Add Task</p>
+            <div>
+                <label>Name</label>
+                <div>100</div><!-- Character limit of text field -->
+                <input type='text'>
+                <small>Maximum of 100 characters.</small>
+            </div>
+            <div>
+                <label>Expected date of completion</label>
+                <input type='text'>
+                <small></small>
+            </div>
+            <div>
+                <label>Status</label>
+                <select>
+                    <option value='to-do'>To-Do</option>
+                    <option value='in-progress'>In Progress</option>
+                    <option value='done'>Done</option>
+                </select>
+                <small></small>
+            </div>
+            <div>
+                <label>Tag</label>
+                <div>30</div><!-- Character limit of text field -->
+                <input type='text'>
+                <small>Tags help to group tasks together</small>
```

Comment:
Why are we using small tags? Is this the correct use case for them?

Reply:
Small tags are for text, they just make the text smaller than the set size.
This way we can have smaller text under each field without having to write a new css class.